documentation/using-hyperview/sensor-management/index.rst

```diff
@@ -0,0 +1,76 @@
+
+Sensor types
+------------
+Every sensor has a type, and there are hundreds of sensor types. The type of the sensor is independent of the sensor category. For example, a power sensor on an asset can be updated by a data collector, computed from available information, or manually updated.
+
+Monitoring profile
+------------------
+Every asset has a monitoring profile, which dictates how asset sensors are updated. The following sensor monitoring profiles are available in Hyperview.
+
+1. BACnet IP
+2. Discovered
+3. Modbus TCP
+4. Manual
+
+The default profile for all assets is Discovered. This means that sensors are automatically discovered and updated.
+
+For Modbus TCP or BACnet IP network-connected assets, a sensor definition is configured in Hyperview and then assigned to the asset as part of the monitoring profile. A data collector is also assigned to update sensor values automatically.
+
+A Manual sensor profile will indicate to Hyperview that the user will manually manage sensors and sensor data.
+
+.. important:: An asset can have only one monitoring profile. Switching an asset's Sensor Monitoring Profile will delete previous sensors and create new ones where applicable. You can manage this setting from the asset's *Information -> Sensor Monitoring* page.
+
+
+=====================
+Sensors and Analyzers
+=====================
+Analyzers are part of the Hyperview expert system. They are called upon to help normalize data, for example, by calculating a power sensor if the asset provides no direct information but provides enough information to calculate it. They are also called upon to calculate computed sensors from information supplied by the user. For example, we can use asset elevation information to calculate operational information like total rack capacity in a location or maximum available contiguous space in a rack to aid in future asset placement.
+
+Analyzers perform a critical role in normalizing the data for assets across manufacturers and models. Furthermore, they perform near real-time analytics on critical metrics, such as energy metering for racks and locations, rack Delta-T, Power Utilization Effectiveness, and Carbon footprint equivalency. The analysis covers the following areas:
+
+1. Power
+2. Energy
+3. Temperature and Humidity
+4. Space utilization
+5. Carbon footprint
+
+Computed sensors are created and associated with the appropriate assets to hold the analysis results. The Hyperview export system monitors the data and event flow and performs analysis in near real-time. Some analytics, such as daily aggregations and statistical summaries, are performed on a schedule.
+
+=====================
+Sensors and licensing
+=====================
+
+Sensors in Hyperview are created through various methods:
+
+1. Asset sensors are automatically discovered and monitored.
+2. Applying a BACnet or Modbus definition to an asset.
+3. Manually created if the asset has a manual sensor monitoring profile.
+4. Computed by an analyzer
```

Comment:
Add a period to the end for consistency.

Reply:
Thanks! updated,